docs/usage.md

```diff
@@ -49,31 +49,32 @@ This version number will be logged in reports when you run the pipeline, so that
 
 ### `-profile`
 
-Use this parameter to choose a configuration profile. Profiles can give configuration presets for different compute environments. Note that multiple profiles can be loaded, for example: `-profile docker` - the order of arguments is important!
+Use this parameter to choose a configuration profile. Profiles can give configuration presets for different compute environments.
```

Comment:
Why is `Note that multiple profiles can be loaded, for example: `-profile docker` - the order of arguments is important!` removed? That is importnant.
```suggestion
Use this parameter to choose a configuration profile. Profiles can give configuration presets for different computing environments.
```

Reply:
This whole section comes directly from the nf-core tools 1.9 template, and was left untouched. https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/%7B%7Bcookiecutter.name_noslash%7D%7D/docs/usage.md#-profile